src/resources/fhirReader.js

```diff
@@ -2,6 +2,35 @@ import {
   format, parse, formatDuration, intervalToDuration,
 } from 'date-fns';
 
+const injectSubType = (resource) => {
+  let subType;
+  switch (resource.resourceType) {
+    case 'Condition':
+    case 'DiagnosticReport':
+    case 'Procedure':
+    case 'Observation':
+      subType = resource.code?.text;
+      break;
+    case 'Encounter':
+      subType = resource.type?.[0]?.text;
+      break;
+    case 'Immunization':
+      subType = resource.vaccineCode?.text;
+      break;
+    case 'MedicationRequest':
+      subType = resource.medicationCodeableConcept?.text;
+      break;
+    case 'CarePlan':
+      subType = resource.category?.[0]?.text;
+      break;
+    default:
+      subType = null;
```

Comment:
A warning here so it's not a bit bucket?

Reply:
yes: a warning for unsupported types.

and/or: make the subType `"Other"`?